src/abstractdataframe/join.jl

```diff
@@ -241,6 +241,11 @@ Join two `DataFrame` objects
   - `:cross` : a full Cartesian product of the key combinations; every
     row of `df1` is matched with every row of `df2`
 
+* `makeunique` : how to handle columns with duplicate names other than `on` in joined tables:
```

Comment:
Can you make this more concise? A bullet list sounds too much for a mere boolean. Why not use a description similar to that used for `names!`?

Also, I know I used it, but I don't think "deduplicate" is a great term for an official documentation. "Make unique" sounds better.

Reply:
OK - fixing